lib/transforms/deployment.js

```diff
@@ -44,7 +40,6 @@ function mapEnvironment({ environment }, isProdEnvironment) {
   const jiraEnv = Object.keys(environmentMapping).find(key => isEnvironment(environmentMapping[key]));
 
   if (!jiraEnv) {
-    app.log.info(`Unmapped environment detected for deployment. Unmapped value is ${environment}. Sending it as unmapped to Jira.`);
```

Comment:
Couldn't access `app` inside this function, so moved logging to use `context` below.